src/librustdoc/html/static/js/search.js

```diff
@@ -4822,7 +4825,7 @@ async function addTab(array, query, display) {
             let alias = " ";
             if (item.is_alias) {
                 alias = ` <div class="alias">\
-<b>${item.alias}</b><i class="grey">&nbsp;- see&nbsp;</i>\
+<b>${item.name}</b><i class="grey">&nbsp;- see&nbsp;</i>\
 </div>`;
```

Comment:
Was wondering if we ever test to make sure this formatting is correct, apparently we do it exactly one in `search-reexport.goml`.

Reply:
Like you mentioned, it's tested in `search-reexport.goml`. :)